### ISSUE TITLE
About integrating ixa-pipe-tok, pos and nerc and using them along with OpenNLP

### ISSUE THREAD

**Issue by Julm** [ixa-ehu/ixa-pipe-nerc#13]
First, thank you for your work on ixa-pipe! 

I'm using ixa-pipe-* in a larger application along with OpenNLP. 

I'd like to upgrade to the latest OpenNLP version but it appears that upgrading uniformeously to OpenNLP 1.8.0 breaks iixa-pipe-nerc (or pos). 

In addition, ixa-pipe-tok and ixa-pipe-pos are not using the same version of the morfologik library from carrot2. (I could isolate the two versions with a jar dependency renaming tool though. But that does not work with OpenNLP because of its object factories)

Thus, a more general question is how should we integrate ixa-pipe-* within an application? And in an application using OpenNLP on its own more specifically.

Thank you very much in advance

**Comment by ragerri** [comment]
Hello,
As you know, OpenNLP 1.8.0 is not backward compatible on some issues. I am working on a new version of IXA pipes and part of the work is to make them compatible with OpenNLP 1.8.0. This will hopefully be ready by the end of July (also the other dependencies issues you commented upon).

Thanks for your interest.

R

**Comment by Julm** [comment]
That is great news! I look forward getting your update.
Thank you so much!
Julien

**Comment by Julm** [comment]
Hi Rodrigo!

Sorry to bother you but when do you think you'll release the new versions of ixa-pipe-pos and ixa-pipe-nerc compatible with OpenNLP 1.8.0?

Thank you so much?
Julien

**Comment by Julm** [comment]
Hi Rodrigo,

Do you still plan to release the OpenNLP 1.8.0 compatible version of Ixa-pipes tools on Maven repo?

Thanks!
Julien

**Comment by ragerri** [comment]
Yes,

but time is limited... :)

I will do it as soon as I can. Sorry for the delay and thanks for the
interest.

Rodrigo

**Comment by Julm** [comment]
Alright, that is good news! :)

And you're welcome, I do think your work deserves it.

Thank you
Julien

**Comment by borice** [comment]
+1 on getting those compatible with latest OpenNLP version.  Appreciate your work!
If you happen to have a rough estimate of when you expect to be able to have these out, it would help us plan whether we wait for this or try to find an alternative way.  We're working on open source software also.

**Comment by ragerri** [comment]
Hello,

I will work on this on January. It should not take me long.

Best,

R

**Comment by Julm** [comment]
Hi there, how is it going? :)

Cheers,
Julien

**Comment by borice** [comment]
Has this project been abandoned? Thanks.

**Comment by ragerri** [comment]
Nope, I am just too busy with a new job, things are hapenning but no
release yet.
